### ISSUE TITLE
Suggestion for "First Steps" instructions - nation/community flow

### ISSUE THREAD

**Issue by ghost** [open-learning-exchange/open-learning-exchange.github.io#239]
![image](https://cloud.githubusercontent.com/assets/20692531/17386211/5a7f727a-59b6-11e6-9940-14f52425790d.png)

Rough draft for nation/community flow.

Dotted lines are "unsure." Dark blue is for database and green is for Crowdin translation. Not sure if better to split this up into two different "layers" or keep it as one diagram.

@DrewPerlman here's how it is in my brain, please correct any errors:
1. Per how I read it from the first steps instructions, I thought that the communities could talk to each other (already), but from ideal best practices, it makes more sense for the nation/master to be the "hub" and the communities the "branches" where hub dictates to branches and branches can talk back to hub in some sort of safe-mode to avoid overriding the master (I probably butchered that explanation from @dogi and @EmilyLarkin :) ).
2. I am guessing the database holds everything, with the nation as the "master" version and ideally immutable, and the communities as the "branch" versions in a json or some other db file format. Therefore all versions of MDWiki pages, info, BeLLs, etc. are stored in parallel in the database.
3. When a community branches off, it can sync with the master/nation first via Internet and then grow/change on its own until it is synced again. Perhaps communities can't upload safely to the nation yet. So nation and new community start off with resources A. Then while community is off by itself collecting resources B, nation can gain resources C. Therefore next time community syncs, it will have resources A+B+C (because you don't want to lose the ones it gained on its own), and nation will have A+C until communities are able to send resources B safely back to nation.

**Comment by rachelpei** [comment]
[flow-chart_nation-community.pptx](https://github.com/open-learning-exchange/open-learning-exchange.github.io/files/400608/flow-chart_nation-community.pptx)
